data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/harm_unit_experience_yes.cfg

```diff
@@ -0,0 +1,135 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [harm_unit]experience=yes
+##
+# Actions:
+# Set all units to a known number of hitpoints.
+# All of these have Alice as the harmer:
+# Use [harm_unit] with Bob as the victim, but a non-fatal amount of damage.
+# Use [harm_unit] with Bob as the victim, a fatal amount of damage but kill=no.
+# Use [harm_unit] with Bob as the victim, a fatal amount of damage.
+# Use [harm_unit] with Charlie and Dave as victims, but a non-fatal amount of damage.
+# Heal Dave.
+# Use [harm_unit] with enough damage to kill Charlie but not Dave.
+##
+# Expected end state:
+# The victims take damage in every step.
+# Alice and Bob get 1 experience point each for the first fight.
+# Alice and Bob get 1 experience point each for the second fight.
+# Alice gets 8 experience points for the third fight, Bob dies.
+# Alice, Charlie and Dave get 1 experience point each for the fourth fight.
```

Comment:
Doesn't match the test.

Just realised that I've documented what I expected it to do from the docs, and coded what I expected it to do from the code. On the next line, I've documented what it really does - Alice gets more XP depending on the number of victims.

Will fix the docs in the next push, but will be AFK for a bit, so waiting for more comments.